### DISCUSSION TITLE
suggestion:  consider using the Conventional Commits specification

### DISCUSSION THREAD

**Discussion by b1nhack** [mfussenegger/nvim-dap#1254] [General]
First of all, thank you so much for developing and maintaining this excellent open-source project. I have greatly benefited from using it.

I would like to make a small suggestion: could you consider using the Conventional Commits specification in the project? This standardized commit message format can help users more easily understand the content of each update, especially in identifying the introduction of new features and important fixes.

Using Conventional Commits has several benefits:

Clear commit history: Developers and users can quickly grasp the purpose of each commit.
Automated changelog generation: Many tools can automatically generate changelogs based on Conventional Commits, saving time and improving accuracy.
Consistent version management: It better aligns with semantic versioning (SemVer), ensuring that version numbers accurately reflect changes.
Enhanced tooling: For instance, lazy.nvim supports Conventional Commits by displaying different font colors based on the significance of the commits, making it visually easier to identify important updates.
If you're interested, here are some resources:

[Conventional Commits Official Documentation](https://www.conventionalcommits.org/)
[Conventional Commits Cheatsheet](https://gist.github.com/Zekfad/f51cb06ac76e2457f11c80ed705c95a3)
Thank you once again for your hard work and dedication!

**Comment by mfussenegger** [comment]
No, this project has release notes for end-users in https://github.com/mfussenegger/nvim-dap/releases

The commits are for developers.
